css/style.css

```
@@ -0,0 +1,596 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "lato", sans-serif;
+}
+
+.hamburger {
+  position: relative;
+  display: block;
+  width: 35px;
+  height: 35px;
+  cursor: pointer;
+  appearance: none;
+  background: none;
+  outline: none;
+  border: none;
+  z-index: 99;
+  margin: 15px;
+}
+
+.hamburger .bar {
+  width: 100%;
+  content: "";
+  display: block;
+  height: 3px;
+  background-color: #272a31;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+
+.hamburger::after {
+  width: 100%;
+  content: "";
+  display: block;
+  height: 3px;
+  background-color: #272a31;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+
+.hamburger::before {
+  width: 100%;
+  content: "";
+  display: block;
+  height: 3px;
+  background-color: #272a31;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+
+.hamburger.is-active::before {
+  transform: rotate(-45deg) translate(-6px, 2px);
+}
+
+.hamburger.is-active::after {
+  transform: rotate(45deg) translate(-10px, -6px);
+}
+
+.hamburger.is-active .bar {
+  opacity: 0;
+}
+
+.mobile-nav {
+  position: fixed;
+  display: block;
+  top: 0;
+  left: 100%;
+  min-height: 100vh;
+  z-index: 98;
+  background-color: #d3d3d3;
+  margin-top: 60px;
+  height: 3px;
+}
+
+ul {
+  text-decoration: none;
+  list-style: none;
+  text-align: center;
+}
+
+.mobile-nav ul {
+  display: block;
+  width: 100%;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  transition: 0.4s;
+  margin: 0;
+  margin-top: 4rem;
+  margin-left: -1rem;
+}
+
+.mobile-nav ul li {
+  padding: 10px 0;
+}
+
+.mobile-nav ul li a {
+  text-decoration: none;
+  color: #272a31;
+  font-family: "lato", sans-serif;
+}
+
+.footer-flex .footer-logo a {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.mobile-nav ul li a:hover {
+  color: #ec5242;
+}
+
+.mobile-nav.is-active {
+  left: 0;
+  width: 100%;
+  margin-top: 0;
+}
+
+@media (min-width: 768px) {
+  .mobile-nav {
+    display: none;
+  }
+
+  .hamburger {
+    display: none;
+  }
+}
+
+@font-face {
+  font-family: Cocogoose;
+  src: url(../fonts/Cocogoose\ Pro-trial.ttf);
+}
+
+@media only screen and (min-width: 768px) {
+  .top-bar-section {
+    display: block;
+    position: relative;
+    background-color: black;
+    padding-right: 10.5rem;
+  }
+
+  .top-bar-container {
+    width: 85%;
+    margin: auto;
+  }
+
+  .top-bar {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  .top-bar li {
+    background-color: black;
+    color: white;
+    padding-inline: 0.8%;
+    cursor: pointer;
+  }
+
+  .top-bar-section .top-bar-container .top-bar li h4 {
```

```suggestion
  .top-bar h4 {
```
This can be used by calling just .top-bar h4
It helps your code to appear neater and easier to read and prevents you from making any typo errors. 
Whenever you have a distinct class select like in this case, you can do without having to call all the class selectors and elements in which it belongs. By typing that whole chain, you may make a typing error. Also, it is more difficult to maintain in case of changes.

I will be making a few guiding suggestions on areas to help write less more readable code that you can easily reuse.